tests/test_rtdc_feat_anc_plugin.py

```diff
@@ -924,14 +948,18 @@ def test_pf_store_non_scalar_plugin_data_with_shape_issue_171():
         events = h5["events"]
         assert "image_gauss_filter" in events.keys()
         feat = h5["events"]['image_gauss_filter']
+        fshape = feat.shape
+        assert len(fshape) == 3 and fshape[1] > 3 and fshape[2] > 3
+        assert b'CLASS' in feat.attrs.keys()
         assert b'IMAGE' in feat.attrs.get('CLASS')
+        assert b'IMAGE_SUBCLASS' in feat.attrs.keys()
 
 
 @pytest.mark.filterwarnings(
     "ignore::dclab.rtdc_dataset.config.WrongConfigurationTypeWarning")
-def test_pf_store_non_scalar_plugin_data_without_shape_issue_171():
-    """Test storage of 3D image plugin feature attributes without shape info
-     in recipe"""
+def test_pf_store_image_plugin_data_without_shape_issue_171():
```

Comment:
Don't link the name of the function to the issue. The function name + short docstring description should be enough to understand what it is doing.

Reply:
It is fixed now.